src/Elliptic/Actions/InitializeSystem.hpp

```diff
@@ -89,16 +89,17 @@ struct InitializeSystem {
                   num_grid_points, 0.};
```

Comment:
Commit message is too long, splits the line in github (probably command line too, but I didn't check)